keyboards/dz60/keymaps/_bonfire/keymap-parts/layers.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * LAYOUT__bonfire                                               Row Keys
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │00 │01 │02 │03 │04 │05 │06 │07 │08 │09 │0a │0b │0c │  0e   │ 14
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │10   │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │1d │1e   │ 14
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+ * │20    │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │2d      │ 13
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┤
+ * │30      │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3d    │3e │ 13
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬┴───┤
+ * │40  │41  │43  │46                      │4a  │4b  │4d  │4e  │ 8
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ *
+ * XXXXXXX = Key does nothing.
+ * _______ = Key that allows the uppermost exposed key in a layer below it.
+*/
+
+#define BASE    0   // Layer: QWERTY / Base
```

Comment:
you could actually make this an enum so you don't actually have to assign a number to each one. This is helpful in the event you want to just sneak in another layer name and don't want to have to redo the numbering. 

```
enum layers{
BASE = 0,
NRMN, 
FCTN, 
KEYB
};